psi4/src/psi4/dcft/dcft_compute_RHF.cc

```diff
@@ -152,11 +152,11 @@ void DCFTSolver::run_simult_dcft_RHF() {
 
     // DIIS on orbitals (AA and BB) and cumulants (AA, AB, BB)
     dpdbuf4 Laa, Lab, Lbb;
-    global_dpd_->buf4_init(&Laa, PSIF_LIBTRANS_DPD, 0, ID("[O,O]"), ID("[V,V]"), ID("[O,O]"), ID("[V,V]"), 0,
+    global_dpd_->buf4_init(&Laa, PSIF_DCFT_DPD, 0, ID("[O,O]"), ID("[V,V]"), ID("[O,O]"), ID("[V,V]"), 0,
```

Comment:
this doesn't need a `psifiles.h` edit?

Reply:
`PSIF_DCFT_DPD` [already exists](https://github.com/psi4/psi4/blob/master/psi4/include/psi4/psifiles.h#L297), so I wouldn't think so.